crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -48,11 +50,26 @@ impl AzureConverter {
     }
 
     fn try_convert(&mut self, input: &MqttMessage) -> Result<Vec<MqttMessage>, ConversionError> {
+        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
         if is_bridge_health(&input.topic.name) {
             Ok(vec![])
+        } else if input.topic.name.starts_with("tedge/health") {
+            let mut payload_json: Map<String, Value> =
```

Comment:
Why not preparing a struct and use `serde` to the struct like this?

```rust
BridgeHeathMessage {
    pid: u32,
    
    status: BridgeStatus,

    #[serde(skip_serializing_if = "Option::is_none")]
    #[serde(with = "time::serde::rfc3339")]
    time: Option<OffsetDateTime>, 
}

#[serde(rename_all = "PascalCase")]
enum BridgeStatus {
    Up,
    Down
}
```

Reply:
We are not going to parse the bridge status to the cloud, so it will be filtered out in line #54. The structure that you are suggesting is for `service` health status right?

Reply:
@PradeepKiruvale Technically yes, but you can apply the same suggested idea to the service message struct...though we have to remember that we should always be preserving extra fields in the message format.

Reply:
Sure, I will do this as part of another ticket where we need to enhance the `health status` message to have `service` information. As of now I just forward the payload as it comes in, without translating.

Reply:
I understood now, as of this PR's scope, having a struct is over-engineering. Once we decided some nice way to report service to az, depending on the decision, we might need a representation of the message model.